graphrag/index/cli.py

```diff
@@ -118,6 +119,20 @@ def index_cli(
             use_kusto_community_reports,
         )
         sys.exit(0)
+    graphrag_config = _read_config_parameters("./",config,progress_reporter)
```

Comment:
I don't understand why this wouldn't be in both. Both during activation & query we need a connection to Cosmos to store & search-on graph related data.